src/Raven.Server/Web/System/DatabasesHandler.cs

```diff
@@ -195,6 +196,25 @@ private Task DbInfo(string dbName)
                     var promotableTask = new PromotableTask(promotable, url, databaseName);
                     nodesTopology.Promotables.Add(GetNodeId(node, topology.WhoseTaskIsIt(promotableTask, ServerStore.IsPassive())));
                 }
+                
+                if (ServerStore.Engine.LeaderTag == ServerStore.NodeTag)
```

Comment:
@aviv86 It means this information will be only available when client connects to leader?